### ISSUE TITLE
"Generate" button for world creation not working with other mods

### ISSUE THREAD

**Issue by skellitor301** [neitsa/PrepareLanding#41]
O.S: Windows 10 10.0.19042 Build 19042 64Bit

RimWorld: 1.3.3071

Problem:
   Likely a mod incompatibility but I'm not sure which. The conflict prevents the Generate button on the Create World window to work, and the error " Exception filling window for RimWorld.Page_CreateWorldParams: System.NullReferenceException: Object reference not set to an instance of an object" pops up. Expanding the error provides the following: 
   
```
Exception filling window for RimWorld.Page_CreateWorldParams: System.NullReferenceException: Object reference not set to an instance of an object
  at PrepareLanding.Patches.PatchCreateWorldParams.Page_CreateWorldParams_CanDoNext (RimWorld.Page_CreateWorldParams& __instance, System.Boolean& __result) [0x0000f] in <2be508a0fde846de982dba70c01c797b>:0 
  at (wrapper dynamic-method) RimWorld.Page_CreateWorldParams.RimWorld.Page_CreateWorldParams.CanDoNext_Patch1(RimWorld.Page_CreateWorldParams)
  at RimWorld.Page.DoBottomButtons (UnityEngine.Rect rect, System.String nextLabel, System.String midLabel, System.Action midAct, System.Boolean showNext, System.Boolean doNextOnKeypress) [0x000e0] in <d89f3c80243544d4a2243b44d67a34bc>:0 
  at RimWorld.Page_CreateWorldParams.DoWindowContents (UnityEngine.Rect rect) [0x004d4] in <d89f3c80243544d4a2243b44d67a34bc>:0 
  at Verse.Window.InnerWindowOnGUI (System.Int32 x) [0x001ce] in <d89f3c80243544d4a2243b44d67a34bc>:0 
UnityEngine.StackTraceUtility:ExtractStackTrace ()
Verse.Log:Error (string)
Verse.Window:InnerWindowOnGUI (int)
UnityEngine.GUI:CallWindowDelegate (UnityEngine.GUI/WindowFunction,int,int,UnityEngine.GUISkin,int,single,single,UnityEngine.GUIStyle)
```

Reproduction steps:
    Mod Manager (unofficial) mod is installed, link to the modlist string provided below.

Logs:
     [Player.log](https://gist.github.com/1441976833b0adef6b1f73990be987f4)

Installed Mods:
    [Link](https://gist.github.com/46431941fda480812ba7757f92fdc3c4)

**Comment by neitsa** [comment]
Thanks a lot for your report! 👍 

This was probably a conflict with a mod, but it's hard to say for sure (although I really do think it was). This should be closed with 10c304566dbdd350635353bf9569e9f11b5d477c.

Already uploaded to steam workshop. From now on there should not be any conflicts left with other mods as I removed all possible source of conflicts.

**Comment by neitsa** [comment]
While I don't think it's possible to have this mode conflict with another mod with the new changes, if you ever stumble upon the same problem, feel free to reopen this issue!
